snapshot/snapshotfs/upload.go

```diff
@@ -690,7 +704,7 @@ func NewUploader(r *repo.Repository) *Uploader {
 	return &Uploader{
 		repo:             r,
 		Progress:         &nullUploadProgress{},
-		IgnoreFileErrors: true,
+		IgnoreFileErrors: false,
```

Comment:
As you pointed out, this is only used for migration ATM.
It makes sense to rename it to `IgnoreReadError` or something similar. Then
* use it also when deriving `ignoreDirErr` as well, not just for file errors;
* update the field documentation describing its intent and effect.

Reply:
ok sounds good

Reply:
Made the change. Did you mean update the kopia end user documentation? It's currently described by the `snapshot migrate` flag and documentation as `Ignore errors when reading source backup`

Reply:
No, I meant adding a comment to the struct field. Most of the time the field names are self documenting, but there are cases in which the usage is subtle and cannot be easily encoded in the field name.

Reply:
Ohh the literal struct field. I totally read that wrong. Upstream PR is open, is it acceptable to push the change directly there or should I handle it separately?